main.go

```diff
@@ -22,17 +23,26 @@ import (
 )
 
 var (
-	pass = color.FgGreen
-	skip = color.FgYellow
-	fail = color.FgHiRed
+	pass   = color.FgGreen
+	skip   = color.FgYellow
+	fail   = color.FgHiRed
+	ignore = flag.Bool("skipnotest", false, "skip packages with no test files")
 )
 
 const paletteEnv = "GOTEST_PALETTE"
 
 func main() {
+	flag.Parse()
 	setPalette()
 	enableOnCI()
-	os.Exit(gotest(os.Args[1:]))
+
+	var startingIndex = 1
+	if strings.HasPrefix(os.Args[1], "-skipnotest") {
+		startingIndex++
+	}
+
+	os.Exit(gotest(os.Args[startingIndex:]))
+
```

Comment:
```suggestion

	os.Exit(gotest(os.Args[1:]))
```

I don't believe this additional logic adds anything to `gotest`. So long as we don't reuse any flag names that `go test` uses, it's fine passing them along.
For example, running `go test --skipnotest ./...` doesn't error and simply ignores the unknown flag.

Reply:
I changed HasPrefix to Contains is that good?